cluster/log-dump/log-dump.sh

```diff
@@ -386,6 +394,12 @@ function dump_nodes() {
   elif [[ ! "${node_ssh_supported_providers}" =~ "${KUBERNETES_PROVIDER}" ]]; then
     echo "Node SSH not supported for ${KUBERNETES_PROVIDER}"
     return
+  elif [[ "${KUBERNETES_CONFORMANCE_PROVIDER:-}" == 'gke' ]] && [[ -n "${NODE_INSTANCE_GROUP[@]:-}" ]]; then
+    for group in "${NODE_INSTANCE_GROUP[@]}"; do
+      node_names+=($(gcloud compute instance-groups managed list-instances \
```

Comment:
does this work in a multi-zone cluster?

Reply:
@msau42  Good question I will try it out. I have only tested with a single zone cluster.
I see [here](https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L57) and [here](https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L592) that 
gke-single-zone-node-instance-group is true by default. We may need to set it to false.